clang/include/clang/Basic/SYCLNativeCPUHelpers.h

```diff
@@ -0,0 +1,7 @@
+#include "clang/Basic/LangOptions.h"
+#include <string>
+namespace clang {
+inline std::string getNativeCPUHeaderName(const LangOptions &LangOpts) {
+  return LangOpts.SYCLIntHeader + ".hc";
```

Comment:
I'm confused. What does LangOpts.SYCLIntHeader hold? I thought it was the name of the integration header? Is the name of the NativeCPU header the same as integration header?

Reply:
Currently the name of the NativeCPU integration header is `<name-of-main-sycl-int-header>.hc`, so we add the `.hc` at the end of the file name